crates/astria-sequencer-relayer/tests/blackbox/helpers/test_sequencer_relayer.rs

```diff
@@ -594,6 +644,25 @@ impl TestSequencerRelayer {
             }),
         );
     }
+
+    /// Mounts a `CometBFT` status response with the chain ID set as per
+    /// `TestSequencerRelayerConfig::sequencer_chain_id`.
+    async fn mount_cometbft_status_response(&self) {
+        use tendermint_rpc::endpoint::status;
+
+        let mut status_response: status::Response = serde_json::from_str(STATUS_RESPONSE).unwrap();
+        status_response.node_info.network =
+            tendermint::chain::Id::from_str(&self.actual_sequencer_chain_id).unwrap();
```

Comment:
```suggestion
            self.actual_sequencer_chain_id.parse().unwrap();
```

Reply:
Done in [458f2a2](https://github.com/astriaorg/astria/pull/1063/commits/458f2a27ff3a3fe97880a3b049c4359c1078058f).